contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -326,6 +326,7 @@ CAST(CASE WHEN Ext.type = 'R' THEN 1 ELSE Ext.is_fixed_role END AS sys.BIT) AS i
 FROM pg_catalog.pg_roles AS Base INNER JOIN sys.babelfish_authid_login_ext AS Ext ON Base.rolname = Ext.rolname
 WHERE pg_has_role(suser_id(), 'sysadmin'::TEXT, 'MEMBER')
 OR Ext.orig_loginname = suser_name()
+OR Ext.orig_loginname = 'jdbc_user'
```

Comment:
`loginname` is not necessarily `jdbc_user` always. We have created this role in init.sh, it could be anything else provided by the customer.

Please don't hardcode this value.